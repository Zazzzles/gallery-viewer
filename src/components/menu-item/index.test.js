import React from 'react';
import renderer from 'react-test-renderer';

import MenuItem from './index';

const collectionProps = {
  title: '',
  user: {
    first_name: '',
    last_name: '',
  },
  preview_photos: [
    {
      urls: {
        thumb: '',
      },
    },
  ],
  total_photos: 0,
};

it('renders correctly', () => {
  const tree = renderer
    .create(<MenuItem active={true} {...collectionProps} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
