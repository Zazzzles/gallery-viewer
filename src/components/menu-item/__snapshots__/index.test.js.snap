// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container Active"
>
  <div
    className="TitleContainer"
  >
    <span
      className="Title"
    >
      
    </span>
    <span
      className="Subtitle"
    >
      
       
      
    </span>
  </div>
  <div
    className="ThumbContainer"
  >
    <div
      className="ThumbImageContainer"
    >
      <img
        alt="Thumb"
        className="ThumbImage"
        src=""
      />
    </div>
    <span
      className="ThumbText"
    >
      + -1 more
    </span>
  </div>
</div>
`;
